handbook/business-operations/README.md

```diff
@@ -947,6 +947,35 @@ To make a request of the business operations department, [create an issue using
 
 > If you're not sure that your request can wait that long, then please ask for urgent help in our group Slack channel: `#g-business-operations`.  Only use this approach or at-mention contributors in business operations directly in urgent situations.  Otherwise, create an issue.
 
+## Contract review
```

Comment:
Written for all three audiences, in order of prescedent: random fleetie, bizops person, general public

```
## Getting a contract signed

Need a signature?  Here's what you do:

### Reviewing a contract
After a reqeust for review is submitted…
```

Reply:
Make it go above all of this:

Rituals
Kanban
Intake
Slack channels